### PR TITLE
docs/vault-secrets-operator: supported openshift versions and OperatorHub options

### PR DESCRIPTION
**Pull request by tvoran**
State the supported OpenShift versions in our docs, and mention the customization options for an operator installed from OperatorHub.

Changed pages:
- https://vault-iaeunj1p1-hashicorp.vercel.app/vault/docs/platform/k8s/vso#supported-kubernetes-distributions
- https://vault-iaeunj1p1-hashicorp.vercel.app/vault/docs/platform/k8s/vso/openshift

### CODE REVIEW

File: website/content/docs/platform/k8s/vso/openshift.mdx [hunk 1]
```diff
@@ -7,14 +7,23 @@ description: >-
...
 
 ## OperatorHub
 
 The Vault Secrets Operator is certified by Red Hat and therefore included in the [OperatorHub section](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.2/html/operators/olm-understanding-operatorhub) of an OpenShift cluster's web console.
 
 Navigate to the OperatorHub page of your OpenShift cluster and search for `Vault Secrets Operator`, then follow the instructions to install.
 
+### Options
+
+Set the following environment variables [on the subscription](https://access.redhat.com/solutions/5761251) to customize the operator's behavior.
```
Comment (schavis) [inline website/content/docs/platform/k8s/vso/openshift.mdx:20]:
```suggestion
Set the following environment variables [on the subscription](https://access.redhat.com/solutions/5761251) to customize the Vault behavior.
```
Vault variables don't customize a person's behavior ;)

**Review by schavis — CHANGES_REQUESTED** [review]

File: website/content/docs/platform/k8s/vso/openshift.mdx (hunk 1 above)
Comment (tomcf-hcp) [reply website/content/docs/platform/k8s/vso/openshift.mdx:20]: This is referring to a [Kubernetes Operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) (Vault Secrets Operator), not a person who is operating Vault.

Comment (tomcf-hcp) [reply website/content/docs/platform/k8s/vso/openshift.mdx:20]: @schavis can we remove this request for change? (see previous comment)

Comment (schavis) [reply website/content/docs/platform/k8s/vso/openshift.mdx:20]: changed the suggestion to clarify the language

**Review by schavis — APPROVED** [review]

**Comment by tvoran** [comment]
Thanks!

**Pull request merged**